scripts/svm/remoteHubPoolPauseDeposits.ts

```diff
@@ -0,0 +1,211 @@
+// This script bridges remote call to pause deposits on Solana Spoke Pool. Required environment:
+// - ETHERS_PROVIDER_URL: Ethereum RPC provider URL.
```

Comment:
NIT: we should think about cleaning these up to be more consistent with other scripts. this is a unique variable that is not named like this in other places. might be ok, for now but as we make these more mature it should standardize.